resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -757,11 +757,14 @@ sqs.message.validation.empty.deduplication_id=Enter a deduplication ID.
 sqs.message.validation.empty.group_id=Enter a group ID.
 sqs.message.validation.empty.message.body=Enter a message body
 sqs.message.validation.long.id=ID exceeds the maximum length of 128 characters.
-sqs.message_table_initial_text=Press "Poll for Messages" to load messages
+sqs.message_table_initial_text=Press "Peek Messages" to load messages
```

Comment:
Could just call it 'View Messages'

Probably want this to actually refer to the button text too rather than repeating 'Peak/Poll/View' messages in this string.